packages/cli/src/workflow-runner.ts

```diff
@@ -177,24 +175,17 @@ export class WorkflowRunner {
 			data.executionMode === 'manual'
 		) {
 			const postExecutePromise = this.activeExecutions.getPostExecutePromise(executionId);
-			postExecutePromise
-				.then(async (executionData) => {
-					try {
-						await this.externalHooks.run('workflow.postExecute', [
-							executionData,
-							data.workflowData,
-							executionId,
-						]);
-					} catch {}
-				})
-				.catch((error) => {
-					if (error instanceof ExecutionCancelledError) return;
-					this.errorReporter.error(error);
-					this.logger.error(
-						'There was a problem running internal hook "onWorkflowPostExecute"',
-						error,
-					);
+			postExecutePromise.catch((error) => {
+				if (error instanceof ExecutionCancelledError) return;
+				this.errorReporter.error(error, {
+					extra: { executionId, workflowId },
 				});
+				this.logger.error('There was an error in the post-execution promise', {
```

Comment:
Do we log this [here](https://github.com/n8n-io/n8n/blob/05858c21532feb1830181f6d9f4ee7f927c5b1a2/packages/core/src/errors/error-reporter.ts#L52) already?

Reply:
that's only when not using Sentry. I can remove this, but this line is not the same as the one on the default error reporter.